src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -156,6 +158,12 @@ void MFEMSidreDataCollection::SetComm(MPI_Comm comm)
   MPI_Comm_rank(m_comm, &myid);
   MPI_Comm_size(m_comm, &num_procs);
 }
+
+void MFEMSidreDataCollection::SetNumFiles(int num_files)
+{
+  SLIC_ASSERT_MSG(num_files > 0, "Output must be to at least 1 file");
```

Comment:
We might want to also check that `num_files` is not larger than `num_ranks`?

But I think that would only be valid after calling `SetComm`, so perhaps that's not possible here.

Reply:
Yes, exactly, that is why I only do that check when we are right about to Save.

I will likely get to making the examples/tests and finishing things next week, thanks for your feedback and sorry for the delay!